src/client/datascience/kernel-launcher/remoteKernelFinder.ts

```diff
@@ -0,0 +1,167 @@
+// This class searches for a kernel that matches the given kernel name.
+// First it searches on a global persistent state, then on the installed python interpreters,
```

Comment:
Ignore my previous suggestion if you saw it before it was deleted.

I believe this file needs to start with 

```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
'use strict';

```

and this comment is continued on line 30 so should move down there

Reply:
Will do